frontend/src/component/common/FilterItem/FilterItemChip/FilterItemChip.tsx

```diff
@@ -40,6 +40,13 @@ const StyledCategoryIconWrapper = styled('div')(({ theme }) => ({
 const StyledOptions = styled('span')(({ theme }) => ({
     color: theme.palette.text.primary,
     fontWeight: theme.typography.fontWeightRegular,
+    textOverflow: 'ellipsis',
```

Comment:
extra protection is 2 explicit items and +n would be too long we need ellipsis. Ellipsis always requires the whiteSpace no wrap, overflow hidden and explicit width.